packages/flutter_tools/test/general.shard/build_system/targets/assets_test.dart

```diff
@@ -55,24 +53,6 @@ flutter:
     expect(globals.fs.file(globals.fs.path.join(environment.buildDir.path, 'flutter_assets', 'assets/wildcard/%23bar.png')).existsSync(), true);
   }));
 
-  test('Does not leave stale files in build directory', () => testbed.run(() async {
```

Comment:
This seems like a good property to have. Why does the test need to be removed?

Reply:
This test requires the whole build system to work. With assets now having new dependencies, it's much more complicated to make it work. @jonahwilliams says its covered elsewhere anyway, but that he'll rework this specific case into a build system check - https://github.com/flutter/flutter/issues/49914 was filed to track that.